terraform/lambda_iam.tf

```diff
@@ -56,6 +56,13 @@ data "aws_iam_policy_document" "binaryalert_analyzer_policy" {
     resources = ["${aws_sns_topic.yara_match_alerts.arn}"]
   }
 
+  statement {
+    sid       = "PublishAlertsToSNS"
```

Comment:
You can't actually have 2 statements with the same `sid`. But you can include the `safe_alerts` topic in the resources in the first statement